doc/articles/features/working-with-xaml-hot-reload.md

```diff
@@ -2,74 +2,91 @@
 
-### Visual Studio 2022
-- Install the [Uno Platform Add-in](https://marketplace.visualstudio.com/items?itemName=unoplatform.uno-platform-addin-2022) from the Visual Studio Marketplace.
-- Create a sample application using the **Uno Platform App** template
-- Build an Uno application head (iOS, Android or WebAssembly), start it (with or without the debugger)
-- Change a XAML file from VS and the app should update
+## How to use Hot Reload
+
+# [**Visual Studio 2022**](#tab/vswin)
+- Setup your environment by following our [getting started guides](xref:Uno.GetStarted.vs2022).
+- Start your application (with or without the debugger, depending on the supported features below)
+- Make changes to your XAML or C# code, save your file then press the red flame icon in the toolbar
+
+# [**Visual Studio Code**](#tab/vscode)
+- Setup your environment by following our [getting started guide](xref:Uno.GetStarted.vscode)
+- Start the application (with or without the debugger, depending on the supported features below)
+- Wait a few seconds for the hot reload engine to become available (see our troubleshooting tips below)
+- Make changes to your XAML or C# code, then save your file
+
+***
+
+## Supported features
+
+# [**Skia**](#tab/skia)
+
+Skia-based targets provide support for full XAML Hot Reload and C# Hot Reload. There are some restrictions that are listed below:
+
+- The Visual Studio 2022 for Windows support is fully available, with and without running under the debugger
+- The VS Code Uno Platform extension does not support Hot Reload when running with a debugger
+- Adding new C# or XAML files to the project is not yet supported
+
+# [**WebAssembly**](#tab/wasm)
+
+WebAssembly is currently providing both full and partial Hot Reload support, depending on the IDE.
+
+- In Visual Studio Code:
+  - Both C# and XAML Hot Reload are fully supported
+  - Adding new C# or XAML files to the project is not yet supported
+- In Visual Studio for Windows:
+  - If your app is using port 5000 in development, [this Visual Studio issue](https://developercommunity.visualstudio.com/t/BrowserLink-WebSocket-is-disconnecting-a/10500228) may cause problems, changing to another port in `Properties/launchSettings.json` (e.g. 5001) will work around it.
+  - [`MetadataUpdateHandlers`](https://learn.microsoft.com/en-us/dotnet/api/system.reflection.metadata.metadataupdatehandlerattribute?view=net-7.0) are invoked without the list of changed types, which means that some hot reload features may not be available.
 
-### Visual Studio Code
-- Follow the [getting started guide](../get-started-vscode.md)
-- Start the application (WebAssembly or Skia)
-- Make modifications to XAML files and save the file
+> [!IMPORTANT]
+> If your solution contains iOS/Android/Catalyst or WebAssembly project heads, Hot Reload will only be available when targeting .NET 8 or later (`net8.0` in the `TargetFrameworks` property)
```

Comment:
should this block be moved out of `#tab/wasm`? or duplicated under `#tab/mobiles`?